egs/wsj/s5/steps/nnet3/get_egs.sh

```diff
@@ -265,23 +293,70 @@ valid_egs_opts="--left-context=$valid_left_context --right-context=$valid_right_
 
 echo $left_context > $dir/info/left_context
 echo $right_context > $dir/info/right_context
-num_pdfs=$(tree-info --print-args=false $alidir/tree | grep num-pdfs | awk '{print $2}')
-if [ $stage -le 3 ]; then
-  echo "$0: Getting validation and training subset examples."
-  rm $dir/.error 2>/dev/null
-  echo "$0: ... extracting validation and training-subset alignments."
 
+if ! $raw_nnet; then
```

Comment:
Agreed. What about steps/nnet3/tdnn/train.sh? Should I create a script like steps/nnet3/tdnn/train_raw.sh?